src/components/dataviz/bar/bar.component.tsx

```diff
@@ -13,14 +13,17 @@ import { MatteTheme } from '../../utilities/createMatteTheme.component';
  */
 // @ts-ignore
 export const Bar: FC<BarSvgProps> = (props) => {
-  const { nivo }: MatteTheme = useTheme();
+  const theme: MatteTheme | null = useTheme();
+
+  const nivo = theme?.nivo;
+
   return (
     <ResponsiveBar
-      colors={nivo?.colorShemes?.mono}
+      colors={typeof nivo !== 'undefined' ? nivo?.colorShemes?.mono : null}
```

Comment:
All other components that use Nivo have also the colors `prop`. Should we also fix it for those components too?

Reply:
I just checked and yes, the other components also cause an error. I'll update the PR 👍